.gitignore

```diff
@@ -22,5 +22,5 @@ public/packs-test/*
 /public/webpack
 js_coverage/
 coverage
-/node_modules
 package-lock.json
+node_modules/esrecurse/package-lock.json
```

Comment:
This one is unnecessary. `node_modules` is entirely being ignored already.

Reply:
Please note the EOF here. 

Check your editor configurations to make sure trim whitespaces and new line on eof are enabled. You may want to configure the [EditorConfig](http://editorconfig.org/) plugin, to make sure your editor always configures it by reading .editorconfig file, which is located in the root folder of this project.